### ISSUE TITLE
Project .xcasset files not being included by Pods-resources.sh

### ISSUE THREAD

**Issue by caplater** [CocoaPods/CocoaPods#3304]
After upgrading to 0.36.0 Pods-resources.sh doesn't seem to include the images in my projects .xcasset files. I can fix / workaround this issue by editing Pods-resources.sh as follows:

Replace

XCASSET_FILES=""

on line 9 with 
XCASSET_FILES="$XCASSET_FILES /Path/To/Images.xcassets"

**Comment by caplater** [comment]
Instead of hard coding the path, it might be better to use:

${PROJECT_DIR}/ProjectName/Images.xcassets

At least that's what I had to use to get XCode Server continuous integration to work.

**Comment by skunkworks** [comment]
I'm seeing the same issue with my project, which includes pods that contain xcassets as well as having its own xcassets.

Our workaround solution in our `Podfile` looks like this:

```
post_install do |installer|
    filename = "Pods/Target Support Files/Pods/Pods-resources.sh"
    text     = File.read(filename)
    File.write(filename, text.gsub(/XCASSET_FILES '\$1'/, "XCASSET_FILES '${PODS_ROOT}/$1'").
                              gsub(/-n \$XCASSET_FILES/, "-n \$\"XCASSET_FILES\"").
                              gsub(/XCASSET_FILES=""/, 'XCASSET_FILES=$(find "${PROJECT_DIR}/ClimateGrowers" -name "*.xcassets" -print)'))
end
```

The first two calls to `gsub` addresses #3247 and #3253. The third addresses this issue.

**Comment by caplater** [comment]
This seems to be a duplicate of #3303 and has been fixed in 0.61

**Comment by kylef** [comment]
Yes, this is a duplicate of #3303, there are also some other issues with xcasset handling which will be fixed by #3263.
